applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx

```diff
@@ -25,10 +27,18 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
   const { formatSource } = useVesselsConnect()
   const onClick = useCallback(() => onVesselClick(vessel), [onVesselClick, vessel])
 
+  const trackEvent = useCallback(() => {
```

Comment:
I don't see this callback being used anywhere, if not used remove it pls

Reply:
Oh I removed the call because now is different with merge feature